### DISCUSSION TITLE
Option to enable/disable Same Price Throughout park by default

### DISCUSSION THREAD

**Discussion by Niels-NTG** [OpenRCT2/OpenRCT2#15698] [Feature requests]
The title explains it basically. I see it as a checkbox under _Options_, tab _Miscellaneous_, [x] _Enabled Same Price Throughout park by default_.

When enabled every new ride placed that has _Same price throughout park_ option under the _Income and costs_ tab enabled by default. Rides that have placed while the option was disabled keep their existing setting.

**Comment by ghost** [comment]
I'd love the same thing as well, but also an option to keep prices between scenarios, actually. It gets pretty tedious with having to set prices on dozens of food+drink stalls over and over, for example.

**Comment by KatieZeldaKat** [comment]
What I really wonder is if players would mind if this checkbox was checked by default without an additional setting being added. I just personally can't imagine having different prices throughout a park for the same items. If the majority of players are like me, it may be better to not have an additional setting in the menu and change the default functionality instead for general ease of access.

**Comment by MASHzj** [comment]
An alternative could be the ability to change default ride prices in a list (in game or in a file). For example, the circus is default 1.50 but that's way too expensive.  A checkbox to set it per park would not help much for me because I almost never build more than 1 circus per park (and that's the case with most ride types).

**Reply by KatieZeldaKat** [reply]
I believe this isn't pertaining to adding new checkboxes to keep the price the same across the park, but to allow the option for the existing ones to be automatically selected. So a specific type of ride (like a circus) wouldn't have a checkbox defined for this, though there can be a box for the on-ride photo or specific items in a stall.

**Reply by ocalhoun6** [reply]
A few of the rides having non-ideal default prices should probably be raised as an entirely separate issue.

The crooked house also comes to mind. And possibly the bumper cars. Oh, and the Ferris wheel.
